index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>My Portfolio</title>
+</head>
+<body>
+  <div id="circle"></div>
+  <header class="header">
+    <a href="#" class="logo">John Do</a>
+    <nav class="nav-flex">
+      <button>Portfolio</button>
+      <button>Contact</button>
+      <button>About</button>
+      <div class="nav-hamburger">
+        <div class="hamburger"></div>
+        <div class="hamburger"></div>
+        <div class="hamburger"></div>
+      </div>
+    </nav>
+  </header>
+  <section class="headline">
+    <h1>Nuri Lacka</h1>
+    <h2>Softwear Developer</h2>
+    <p>
```

Comment:
- [ ] (Optional) Nice work done in the headline section. I would like to suggest that you correct the typographical error made. The correct spelling is `software` and not `softwear`